pkg/sat/loader.go

```diff
@@ -27,7 +28,7 @@ func NewLoader() *Loader {
 		m: &Model{
 			packages:                    map[string][]*Var{},
 			vars:                        map[string]*Var{},
-			bestPackages:                map[string]*api.Package{},
+			bestPackages:                map[BestKey]*api.Package{},
```

Comment:
why do we want to key this using arch?  Currently this picks the best version of a package and when we key by arch we will have Foo.i686 and Foo.x86_64 as separate packages in this list.  Why not just prune the additional arch packages at this point?